servicetests/test_cases/test_jobs_execution.py

```diff
@@ -0,0 +1,98 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0.
+
+import argparse
+import json
+import os
+import sys
+import uuid
+
+import boto3
+
+import run_in_ci
+import ci_iot_thing
+
+
+def main():
+    argument_parser = argparse.ArgumentParser(
+        description="Run Jobs test in CI")
+    argument_parser.add_argument(
+        "--input-uuid", required=False, help="UUID for thing name. UUID will be generated if this option is omit")
+    argument_parser.add_argument(
+        "--region", required=False, default="us-east-1", help="The name of the region to use")
+    argument_parser.add_argument(
+        "--mqtt-version", required=True, choices=[3, 5], type=int, help="MQTT protocol version to use")
+    parsed_commands = argument_parser.parse_args()
+
+    try:
+        iot_client = boto3.client('iot', region_name=parsed_commands.region)
+        secrets_client = boto3.client("secretsmanager", region_name=parsed_commands.region)
+    except Exception as e:
+        print(f"ERROR: Could not make Boto3 iot-data client. Credentials likely could not be sourced. Exception: {e}",
+              file=sys.stderr)
+        return -1
+
+    current_path = os.path.dirname(os.path.realpath(__file__))
+    cfg_file_mqtt_version = "mqtt3_" if parsed_commands.mqtt_version == 3 else "mqtt5_"
+    cfg_file = os.path.join(current_path, cfg_file_mqtt_version + "jobs_cfg.json")
```

Comment:
I feel I'd rather have the config file as an argument, so that we could possibly add more tests in the future. Otherwise, whenever we add a new test, we would need update the script here.

Reply:
That's a good point, I'll do this